SPECS/python-ssh2-python/python-ssh2-python.spec

```diff
@@ -0,0 +1,138 @@
+%{?python_enable_dependency_generator}
+%global debug_package %{nil}
+
+%global modname ssh2-python
+
+Name:           python-%{modname}
+Version:        0.27.0
+Release:        1%{?dist}
+Summary:        Super fast SSH library - bindings for libssh2
+License:        LGPLv2+
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://github.com/ParallelSSH/ssh2-python
+Source0:        %{url}/archive/refs/tags/%{version}.tar.gz#/ssh2-python-%{version}.tar.gz
+Source1:        conftest.py
+BuildRequires:  cmake
+BuildRequires:  gcc
+BuildRequires:  libssh2-devel
+BuildRequires:  python3-devel
+BuildRequires:  python3-setuptools
+BuildRequires:  python3-Cython
+BuildRequires:  python3-sphinx
+BuildRequires:  python3-sphinx_rtd_theme
+
+%if %{with_check}
+BuildRequires:  python3-pytest
+BuildRequires:  %{_sbindir}/sshd %{_bindir}/ssh-agent
+%endif
+
+%description
+%{summary}.
+
+%package -n python3-%{modname}
+Summary:        %{summary}
+%{?python_provide:%python_provide python3-%{modname}}
+
+%description -n python3-%{modname}
+%{summary}.
+
+Python 3 version.
+
+%prep
+%autosetup -n %{modname}-%{version}
+# No bundled libs
+rm -vrf libssh2
+sed -i -r 's:build_ssh2[(].*:pass:' setup.py
+# Remove pre-generated sources
+rm $(grep -rl '/\* Generated by Cython')
+
+%build
+export HAVE_AGENT_FWD=0
+# use build_ext to completely instruct cythonize options
+export CFLAGS="${CFLAGS:-${RPM_OPT_FLAGS}}"
+export LDFLAGS="${LDFLAGS:-${RPM_LD_FLAGS}}"
+export CXXFLAGS="${CXXFLAGS:-${RPM_OPT_FLAGS}}"
+%{__python3} setup.py build_ext --inplace
+%{__python3} -msphinx -M html doc _build
+
+%install
+%py3_install
+
+%check
+# FIXME skip b0rken tests with segmentation fault from jinja2, rhbz#2007478
+rm tests/test_knownhost.py tests/test_session.py\
+ tests/test_sftp.py tests/test_utils.py
+# FIXME skip b0rken tests with json
+rm tests/test_exceptions.py
+# FIXME skip another b0rken tests
+rm tests/test_channel.py
+# disable some options for sshd running inside mock, see tests of libssh2
+echo UsePrivilegeSeparation no >> tests/embedded_server/sshd_config.tmpl
+echo StrictModes no >> tests/embedded_server/sshd_config.tmpl
```

Comment:
I'm not certain we need these, as we do not primarily build in mock. Can you run these package tests without these configurations, and see if they're still necessary?

Reply:
No, they are not needed. I have removed them.